js/scripts.js

```diff
@@ -10,5 +10,5 @@ if (value > 0) {
 } else if (value == NaN) {
```

Comment:
`isNaN(value)` will be better ;)

`https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/isNaN`